xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
Ahh great!  Then we can have a different test.  Our `-i`, `-l` and `-c` flags all come from Bash, etc. for future reference.